react/features/invite/actionTypes.js

```diff
@@ -41,6 +41,16 @@ export const REMOVE_PENDING_INVITE_REQUESTS
  */
 export const SET_CALLEE_INFO_VISIBLE = 'SET_CALLEE_INFO_VISIBLE';
 
+/**
+ * The type of Redux action to set the visibility of the dial in summary.
+ *
+ * {
+ *     type: SET_DIAL_IN_SUMMARY_VISIBLE,
+ *     visible: boolean
+ * }
+ */
+export const SET_DIAL_IN_SUMMARY_VISIBLE = 'SET_DIAL_IN_SUMMARY_VISIBLE';
```

Comment:
(Not necessarily applying to this PR but still). This functionality is not really related to inviting others to the meeting. Maybe we need a new one.

Reply:
I agree, but I saw that the web implementation of this component was here, so I wanted to stay consistent.